docs/cas-server-documentation/installation/Logout-Single-Signout.md

```diff
@@ -83,6 +72,25 @@ A sample SLO message:
 </samlp:LogoutRequest>
 ```
 
+### Front Channel
+
+CAS issues asynchronous AJAX `GET` logout requests via `JSONP` to authenticated services.
+The expected behaviour of the CAS client is to invalidate the application web session. 
+
+Support for front channel logout calls may be available only in development version of CAS clients.
```

Comment:
Please take out this line (#80)

Reply:
Or put in a different way: Front channel logout may not be available for all CAS clients. Ensure your CAS client does support this behavior before trying it out.